client/common/web3Storage.ts

```diff
@@ -0,0 +1,5 @@
+import { Web3Storage } from 'web3.storage';
+
+export const makeStorageClient = () => {
+  return new Web3Storage({ token: process.env.WEB3STORAGE_TOKEN ?? '' });
```

Comment:
Currently this token will be exposed in our front-end bundle, meaning any malicious actor can take this and put an infinite amount of files. Do we want to make this part of user-retrieval, ...? Or is this token scoped to only read?

Reply:
Hi @JoviDeCroock, you're completely right about it being available to malicious actors. How can i not expose the token? I was not aware that it would be exposed in our front end bundle.

Reply:
We can do this as a follow-up issue as we're not live yet but I'd suggest incorporating this in our `api-routes` so `/ap/upload-avatar` which will stream fe --> api-route --> web3.storage this makes our tokens safely stored in a serverless function and away from our users

Reply:
https://nextjs.org/docs/api-routes/introduction

Reply:
Created #143